exp/controllers/resource_reconciler.go

```diff
@@ -100,6 +103,36 @@ func (r *ResourceReconciler) Reconcile(ctx context.Context) error {
 	return nil
 }
 
+// Pause pauses reconciliation of the specified resources.
+func (r *ResourceReconciler) Pause(ctx context.Context) error {
```

Comment:
I don't see an equivalent general purpose resource pause method in the v1beta1 APIs. Is this new approach something we'll bring into the v1 APIs (or am I missing something?)

Reply:
This mechanism was modeled after what we do for the v1beta1 apis. #3735 was the first PR which added that pattern.

Reply:
I think in the end I'm just observing that the general gvk resource scaffolding we have here doesn't exist in the current API controllers, and that makes sense as we don't need this for converting back and forth between native ASO and raw JSON.